CONTRIBUTING.md

```diff
@@ -1,5 +1,8 @@
-For information about how to contribute to the Void documentation, please read the following sections of the Void Handbook:
+For information about how to contribute to the Void documentation, please read
+the following sections of the Void Handbook:
 
 - [About this Handbook](https://docs.voidlinux.org/about-handbook/index.html)
-- [Style Guide](https://docs.voidlinux.org/contributing/void-docs/style-guide.html)
-- [Submitting Changes](https://docs.voidlinux.org/contributing/void-docs/submitting.html)
+- [Style
+   Guide](https://docs.voidlinux.org/contributing/void-docs/style-guide.html)
+- [Submitting
+   Changes](https://docs.voidlinux.org/contributing/void-docs/submitting.html)
```

Comment:
`vmdfmt` doesn't need to be run for this file, I don't think this needs to be in this PR, right?